pkg/pillar/cmd/downloader/syncop.go

```diff
@@ -0,0 +1,272 @@
+package downloader
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"net/url"
+	"os"
+	"time"
+
+	zconfig "github.com/lf-edge/eve/api/go/config"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/lf-edge/eve/pkg/pillar/zedUpload"
+	"github.com/lf-edge/eve/pkg/pillar/zedcloud"
+	log "github.com/sirupsen/logrus"
+)
+
+// Drona APIs for object Download
+func handleSyncOp(ctx *downloaderContext, key string,
+	config types.DownloaderConfig, status *types.DownloaderStatus,
+	dst *types.DatastoreConfig) {
+	var (
+		err                                        error
+		errStr, locFilename, remoteName, serverURL string
+		syncOp                                     zedUpload.SyncOpType = zedUpload.SyncOpDownload
+		trType                                     zedUpload.SyncTransportType
+		auth                                       *zedUpload.AuthInput
+	)
+
+	if status.ObjType == "" {
+		log.Fatalf("handleSyncOp: No ObjType for %s\n",
+			status.Safename)
+	}
+
+	log.Debugf("handleSyncOp: config: %+v", config)
+	log.Debugf("handleSyncOp: IsContainer: %v", config.IsContainer)
```

Comment:
We can remove this.. This will be printed as part of config anyways..

Reply:
Sure. Removing.